src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -53,12 +53,32 @@ public void storeMessage(Message message) {
    *     message. List is sorted by time descending.
    */
   public List<Message> getMessages(String user) {
+    return messageHelp(user);
+  }
+
+  /**
+   * Get messages posted by every user.
+   * 
+   * @return a list of messages posted by all users of the site, or an empty list if
+   *    there are no messages
+   */
+  public List<Message> getAllMessages() {
+    return messageHelp(null);
```

Comment:
What was your reason for null instead of an empty string here?

Either way I think you should pull it out into a constant to give it context as null can mean anything, giving the variable you are passing to the method a name can help increase readability a lot.